tests/unit/fsattrs_test.c

```diff
@@ -0,0 +1,46 @@
+#include <test.h>
+#include <cmockery.h>
+#include <fsattrs.h>
+#include <stdlib.h>
+
+
+static void test_file_immutable_flag(void)
+{
+    /* Get immutable flag from non existent file */
+    unlink("no_such_file_639123");
+    bool flag;
+    FSAttrsResult res = FileGetImmutableFlag("no_such_file_639123", &flag);
```

Comment:
If this returns `FS_ATTRS_NOTSUPP` is it safe to assume that this stuff is not supported and the rest of the test is irrelevant? We could just return early here?

Reply:
Good point :rocket: